src/Bridges/Constraint/bridges/fix_parametric_variables.jl

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    FixParametricVariablesBridge{T,S} <: AbstractBridge
```

Comment:
docs do not make it clear if it handles constant * parameter * variable and constant * parameter

Reply:
Updated. It handles only `constant * parameter * variable`, leaving `constant * parameter` untouched.